isala.local/app/controllers/login.php

```diff
@@ -143,13 +156,47 @@ protected function attemptAuthenticate($uid, $tfa)
             && !$this->model->getLDAP()->query('uidExists', [$uid, "account"])
         ) return false;
 
+        // Check if ip is blocked
+        $blocked_ip_arr = $this->model->getDB()->query('blockedIPArray', [$uid]);
+        if (count($blocked_ip_arr) > 0) {
```

Comment:
Copy/paste misgegaan? zie twee keer dat er gekeken wordt of het Ip is geblokkeerd

Reply:
De ene is Login, de andere 2FA